src/renderer/extensions/vueNodes/composables/useNodePointerInteractions.ts

```diff
@@ -52,6 +52,16 @@ export function useNodePointerInteractions(
       return
     }
 
+    const stopNodeDragTarget = (event.target as HTMLElement | null)?.closest(
+      '[data-capture-node="true"]'
+    )
+    if (stopNodeDragTarget) {
+      if (!shouldHandleNodePointerEvents.value) {
+        forwardEventToCanvas(event)
+      }
```

Comment:
Just commenting for signal: I still really don't like the pattern of decorating components and shuffling events and synthetic events around.